src/libs/KeyboardShortcut/index.js

```diff
@@ -90,13 +90,15 @@ const KeyboardShortcut = {
      * @param {Function} callback The callback to call
      * @param {String|Array} modifiers Can either be shift or control
      * @param {Boolean} captureOnInputs Should we capture the event on inputs too?
+     * @returns {Function} clean up method
      */
     subscribe(key, callback, modifiers = 'shift', captureOnInputs = false) {
         const keyCode = this.getKeyCode(key);
         if (events[keyCode] === undefined) {
             events[keyCode] = [];
         }
         events[keyCode].push({callback, modifiers: _.isArray(modifiers) ? modifiers : [modifiers], captureOnInputs});
+        return () => this.unsubscribe(key);
```

Comment:
This is kind of extra. Probably didn't need to do it, but I like this pattern for cleaning up subscriptions.